tests/structure/test_charges.py

```diff
@@ -0,0 +1,308 @@
+)
+acetaldehyde.bonds = BondList(
+    acetaldehyde.array_length(),
+    np.array([[0,1], [1,2], [0,3], [0,4], [0,5], [1,6]])
+)
+
+
+acetone = array(
+    [carbon, carbon, carbon, oxygen, hydrogen, hydrogen, hydrogen,
+    hydrogen, hydrogen, hydrogen]
+)
+acetone.bonds = BondList(
+    acetone.array_length(),
+    np.array([[0,1], [1,2], [1,3], [0,4], [0,5], [0,6], [2,7], [2,8],
+    [2,9]])
+)
+
+
+hydrogen_cyanide = array(
+    [carbon, nitrogen, hydrogen]
+)
+hydrogen_cyanide.bonds = BondList(
+    hydrogen_cyanide.array_length(),
+    np.array([[0,1], [0,2]])
+)
+
+
+acetonitrile = array(
+    [carbon, carbon, nitrogen, hydrogen, hydrogen, hydrogen]
+)
+acetonitrile.bonds = BondList(
+    acetonitrile.array_length(),
+    np.array([[0,1], [1,2], [0,3], [0,4], [0,5]])
+)
+
+# For this purpose, parametrization via pytest is performed
+@pytest.mark.parametrize("molecule, expected_results", [
+    (methane, (-0.078,)),
+    (ethane, (-0.068, -0.068)),
+    (ethylene, (-0.106, -0.106)),
+    (acetylene, (-0.122, -0.122)),
+    (fluoromethane, (0.079,)),
+    (difluoromethane, (0.23,)),
+    (trifluoromethane, (0.38,)),
+    (tetrafluoromethane, (0.561,)),
+    (fluoroethane, (0.087, -0.037)),
+    (trifluoroethane, (0.387, 0.039)),
+    (methanole, (0.033,)),
+    (dimethyl_ether, (0.036, 0.036)),
+    (formaldehyde, (0.115,)),
+    (acetaldehyde, (-0.009, 0.123)),
+    (acetone, (-0.006, 0.131, -0.006)),
+    (hydrogen_cyanide, (0.051,)),
+    (acetonitrile, (0.023, 0.06))
+])
+def test_partial_charges(molecule, expected_results):
+    """
+    Test whether the partial charges of the carbon atoms comprised in
+    the molecules given in table 3 of the publication computed in this
+    implementation correspond to the values given in the publication
+    within a certain tolerance range.
+    """
+    charges_ = charges.partial_charges(molecule)
+    assert charges_[molecule.element == "C"].tolist() == \
+        pytest.approx(expected_results, abs=1e-2)
+
+
+@pytest.mark.parametrize("molecule", [
+    methane,
+    ethane,
+    ethylene,
+    acetylene,
+    fluoromethane,
+    difluoromethane,
+    trifluoromethane,
+    tetrafluoromethane,
+    fluoroethane,
+    trifluoroethane,
+    methanole,
+    dimethyl_ether,
+    formaldehyde,
+    acetaldehyde,
+    acetone,
+    hydrogen_cyanide,
+    acetonitrile
+])
+def test_total_charge_zero(molecule):
+    """
+    In the case of the 17 molecules given in table 3, it is verified
+    whether the sum of all partial charges equals the sum
+    of all formal charges (in our case zero since we are exclusively
+    dealing with uncharged molecules).
+    """
+    total_charge = np.sum(charges.partial_charges(molecule))
+    assert total_charge == pytest.approx(0, abs=1e-15)
+
+
+# Test whether a formal charge of +1 has the expected effects on
+# carbons's partial charge in methane
+# AtomArray with positive carbon must be created ab inition in order to
+# prevent influencing 'methane'
```

Comment:
This comment is more or less redundant to the docstring of `test_pos_formal_charge()`. So I think it can be removed.

Reply:
Indeed.

Reply:
Indeed.